examples/landau-zener-stuckelberg.ipynb

```diff
@@ -91,58 +91,46 @@
    "source": [
     "# ODE settings (for list-str format)\n",
     "options = Options()\n",
-    "options.rhs_reuse = True\n",
     "options.atol = 1e-6 # reduce accuracy to speed\n",
-    "options.rtol = 1e-5 # up the calculation a bit"
+    "options.rtol = 1e-5 # up the calculation a bit\n",
+    "options.rhs_reuse = True # Compile Hamiltonian only the first time."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 7,
    "metadata": {},
    "outputs": [],
-   "source": [
-    "# for function-callback style time-dependence\n",
-    "def hamiltonian_t(t, args):\n",
-    "    \"\"\" evaluate the hamiltonian at time t. \"\"\"\n",
-    "    H0 = args[0]\n",
-    "    H1 = args[1]\n",
-    "    w  = args[2]\n",
-    "    return H0 + H1 * np.sin(w * t)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "# perform the calculation for each combination of eps and A, store the result\n",
     "# in a matrix\n",
     "def calculate():\n",
     "\n",
     "    p_mat = np.zeros((len(eps_list), len(A_list)))\n",
-    "\n",
-    "    pbar = ProgressBar(len(eps_list))\n",
     "    \n",
+    "    H0 = - delta/2.0 * sx \n",
+    "    \n",
+    "    H1 = - sz/2 \n",
+    "    # Function method\n",
+    "#     H1 = [H1, lambda t, args: args['eps'] ]\n",
+    "    # String method\n",
+    "    H1 = [H1, 'eps']\n",
```

Comment:
```suggestion
    "    # Define H1 (first time-dependent term)\n",
    "    # String method:\n",
    "    H1 = [- sz / 2, 'eps']\n",
    "    # Function method:\n",
    "    # H1 = [- sz / 2, lambda t, args: args['eps'] ]\n",
```
Suggestion for formatting the example ^^^. I like this because it puts the simple clear way first and avoids the confusing `H1 = [H1, ...]` redefinition of H1. I also tided up the indentation and spaces after `#`.

If you agree that it looks nicer, could you do the same for H2?